crates/gpui/src/platform/linux/x11/window.rs

```diff
@@ -197,6 +199,18 @@ impl X11WindowState {
                 &[atoms.WM_DELETE_WINDOW],
             )
             .unwrap();
+        let transparent = params.window_background != WindowBackgroundAppearance::Opaque;
+        if transparent {
+            xcb_connection
+                .change_property32(
+                    xproto::PropMode::REPLACE,
+                    x_window,
+                    atoms._NET_WM_WINDOW_OPACITY,
```

Comment:
This appears to be an alpha multiplier that the compositor applies to the whole window rather than a hint that the window has an alpha channel. From what I've found the latter needs to be done through the XRender extension by finding a visual that supports alpha.

Reply:
thanks for the info! removing now
I do have the visual selection for X11, but it doesn't work correctly yet